public/collections.json

```diff
@@ -1,4 +1,28 @@
 {
+  "Generative AI": {
+    "id": "genAI",
+    "languages": ["en"],
+    "jurisdictions": ["EU"],
+    "maintainers": [
+      {
+        "name": "Open Terms Archive",
+        "url": "https://opentermsarchive.org"
+      },
```

Comment:
I'm not sure about that at this stage… We significantly contribute indeed, but then we could be added to many other collections as well. I'm in favour of removing Open Terms Archive and consider that we are supporters of every collection anyway.
```suggestion
```